components/Nav/SidebarMenus.js

```diff
@@ -101,4 +103,66 @@ const SidebarMenu = ({ onRouteChange }) => (
   </MenuInner>
 )
 
-export default (SidebarMenu)
+function getSectionPath(parentPathname, title) {
+  return `${parentPathname || ''}#${titleToDash(title)}`
+}
+
+function isFolderOpen(currentHref, { pathname, title, sections }) {
+  return (
+    sections.reduce((sum, v) => (
+      sum || window.location.href.endsWith(getSectionPath(pathname, v.title))
+    ), false) ||
+    window.location.href.endsWith(pathname || '#' + titleToDash(title))
```

Comment:
Hi, @morajabi! Maybe it's better to use a template string in such kind of places in this PR?

Reply:
Actually I used template strings in this file but I think adding a single # to the beginning of another string is simple enough to be done this way. 
`` `#${titleToDash(title)}` `` vs `` '#' + titleToDash(title) ``